### ISSUE TITLE
Building individual applications

### ISSUE THREAD

**Issue by cosarara97** [Rakashazi/emu-ex-plus-alpha#30]
Following the instructions, I managed to build the imagine library. My goal, however, is building GBC.emu, for which there are no instructions.

Here's the (cleaned) log of what I did: http://pastebin.com/mWtjHMPU

**Comment by thededman** [comment]
Not sure when Rakashazi will answer, so I will try to help.

So you have compiles the imagine framework, great. 
you did the make -f $IMAGINE_PATH/makeAll-Android.sh
This is the completed without errors go to the IMAGINE directory, then run android-release.mk

make -f android-release.mk install v=1 -j4 (this will install the library for the build)
-----------------------------
You also need to compile and install the EmuFrameWork.
In this directory- make -f android-release.mk install v=1 -j4 
*In my virtual mac OS X I have to run this command twice to get the libemuframework.a to install.
If no whammies here, then you can NOW build your emulator.
------------------------------
I would have a better answer but can figure out Android framework setup. 
Hope this helped or you'll have to wait for Rakashazi to help you out.

**Comment by cosarara97** [comment]
EmuFramework's make fails:

    cp: cannot stat ‘lib/android-9-armv7-release/emuframework.pc’: No such file or directory

**Comment by thededman** [comment]
I have seen this issue before in Mac OS X. One method with to locate the emuframework.pc and copy to the lib folder that it wanted to be in and re-run the command. 
The other issues I have was not using the proper commands. type "make -f andriod-release.mk install v=1 -j4 TAR=gnutar" 
As typing this, I had flushed my TMP and imagine-sdk folder. Then rebuilt. On the EmuFramework I got an error, re-run command and completed. 
I was able to compile the GBA.EMU and deliver to my iPad. 
 *Not sure how this is going to be on linux. Which linux OS are you using? I may find sometime to try to build a Android version for knowing how in linux.
